README.md

```diff
@@ -226,7 +234,7 @@ More examples are available under `examples` directory.
 
 To use the same port for custom HTTP handlers (e.g. serving `swagger.json`),
 gRPC-gateway, and a gRPC server, see
-[this code example by CoreOS](https://github.com/philips/grpc-gateway-example/blob/master/cmd/serve.go)
+[example by CoreOS](https://github.com/philips/grpc-gateway-example/blob/master/cmd/serve.go)
```

Comment:
This stutters a little now, how about this?
```suggestion
[this example by CoreOS](https://github.com/philips/grpc-gateway-example/blob/master/cmd/serve.go)
```

Reply:
Thanks for the correction. @johanbrandhorst 😄